core/stats/stats.go

```diff
@@ -80,10 +80,10 @@ func (s *Stats) GetUptime() time.Time {
 	return s.uptime
 }
 
-func (s Stats) Height() uint64 {
+func (s *Stats) Height() uint64 {
 	return s.Blockchain.Height()
 }
 
-func (s Stats) BlockHash() string {
+func (s *Stats) BlockHash() string {
 	return s.Blockchain.Hash()
```

Comment:
Why change them to pointers? They are getters it's unnecessary?

Reply:
Since we are returning copied data, it's safe to use a point receiver. It avoids unnecessary copies of the struct. So I shift it.